src/ui/components/filters/filterboxcomponent.js

```diff
@@ -259,7 +260,6 @@ export default class FilterBoxComponent extends Component {
     if (searchOnChange) {
       this._search();
     }
-    onChange && onChange();
```

Comment:
I think you need to update the onFilterNodeChange method signature now that this is removed.

Reply:
my bad, updated